recipes/meta/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "meta" %}
```

Comment:
isuruf asked that you change the name to meta-neuro earlier citing concerns that `meta` was too generic. You agreed, but you gave no reason why you reverted back to `meta`.

is it possible to revert back to `meta-neuro`?

Reply:
I built my package at bioconda with meta, and I would like to be consistent with other package managers. https://bioconda.github.io/recipes/meta/README.html

Reply:
The problem with this statement is that you never said it BEFORE you reverted the changes that Isuru suggested.

its a little hard to tell who is who, but Isuru is very knowledgeable on this front.

Given he expressed prior concerns about this, I will have to wait for him to respond again.

Reply:
The main issue that Isuru is raising is that we also try to follow pypi package naming: https://pypi.org/project/meta/ so the name `meta` clashes quite hard.